src/qbe_ir_generator.cpp

```diff
@@ -203,27 +227,27 @@ class OpNameGetter::OpNameGetterImpl : public NonModifyingVisitor {
   }
 
   void Visit(const GreaterThanExprNode&) override {
-    op_name_ = "sgt";
+    op_name_ = "csgtw";
   }
 
   void Visit(const GreaterThanOrEqualToExprNode&) override {
-    op_name_ = "sge";
+    op_name_ = "csgew";
```

Comment:
What is the reasoning behind this update?
My assumption is that we missed the `c` prefix, making `sge` itself an invalid op.
Additionally, is the `w` suffix indicating integer ("word") type?
Am I correct in this interpretation?

Reply:
Yep. The correct comparison instructions are in this section[1].

[1] https://c9x.me/compile/doc/il.html#Instructions-Index

Reply:
Understand. I suggest using the verb `Fix` instead of `Update` in the commit message to tell better that the original is ill-formed or invalid 👍 .

Reply:
Done! Great suggestion. 💯